about.html

```diff
@@ -0,0 +1,124 @@
+    <link rel="shortcut icon" href="./app/assets/head.png" type="image/x-icon" />
+    <script
+      src="https://kit.fontawesome.com/bc94a42a6b.js"
+      crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap"
+      rel="stylesheet" />
+    <link rel="stylesheet" href="dist/style.css" />
+    <title>Smart | About Us</title>
+  </head>
+  <body>
+    <div class="container-about">
+      <!-- ================== Header ====================== -->
+
+      <header class="header-top flex flex-jc-b flex-center limit">
+        <p class="logo flex flex-center">
+          <img src="./app/assets/head.svg" alt="" />
+          <a href="index.html">Sm<span class="span">art</span></a>
+        </p>
+        <nav class="menu flex flex-al-c flex-jc-c">
+          <ul class="nav-links flex flex-jc-b">
+            <li><a class="nav-link" href="index.html">Home</a></li>
+            <li><a class="nav-link" href="about.html">About</a></li>
+            <li><a class="nav-link" href="#program">Program</a></li>
+            <li><a class="nav-link" href="#join">Join</a></li>
+            <li><a class="nav-link" href="#sponsor">Sponsor</a></li>
+            <li><a class="nav-link contact" href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+
+        <div class="btn-container flex flex-center">
+          <span class="menu-btn"></span>
+        </div>
+      </header>
+
+      <!-- ================== Hero Section ====================== -->
+
+      <section class="hero-about flex flex-col flex-al-c limit">
+        <header class="about-text">
+          <p class="hero-about-desc">"Sharing our passion for learning with you."</p>
+          <h1 class="hero-about-title">Learn With Us</h1>
+        </header>
+        <div class="about-desc">
+          <p class="first">
+            Providing an easy acces to the best online learning materials is our first
+            duty. At smart, we believe that all people can bring out the best of
+            themselves when provided with the rigt opportunity to aquire knowledge but
+            also implement this one. We provide students all over the world with
+            everything you need to study anytime, anywhere. We put great emphasis on
+            maximum flexibility in our study models to help you fit learning to your
+            schedule.
+          </p>
+
+          <p class="second">
+            Please contact us for any further questions at
+            <a href="mailto: questions@smart.com">questions@smart.com</a>
+          </p>
+        </div>
+      </section>
+
+      <!-- ================== Logo ====================== -->
+
+      <section class="logo-about flex flex-col flex-center limit">
+        <header class="logo-header">
+          <h1>Smart Online School Logo</h1>
```

Comment:
- [ ] Please note that it is SEO best practice to use only one `<h1>` tag for every page of your web app. The `<h1>` tag should contain the title of your page so that screen readers can easily identify what your page is about from the title. Please check [here](https://ahrefs.com/blog/h1-tag/#:~:text=tags%20improve%20accessibility-,H1%20tags%20helps%20search%20engines%20understand%20the%20page,what%20my%20page%20is%20about.%E2%80%9D) for more details.